### PR TITLE
[FRONT] #382 navigation drawer ui enhancements

### PR DESCRIPTION
**Pull request by ntchou**
- Fix menu lang translation (greek)
- minify svg logo

Screenshot_2020-12-28_14-33-11
Screenshot_2020-12-28_14-33-44
Screenshot_2020-12-28_14-33-56

### CODE REVIEW

File: src/components/Logo/Logo.tsx [hunk 1]
```diff
@@ -1,32 +1,37 @@
...
+        width: 181px;
+        height: 66px;
+    }
+`
 
 function Logo() {
     const { t } = useTranslation()
-    const isMobile = useDetectMobileOrTablet()
-    let logoToChoose = ''
+    const isMobile = isIos() || isAndroid()
```
Comment (MattiaPrimavera) [inline src/components/Logo/Logo.tsx:24]: I think that is not exactly the same as `useDetectMobileOrTablet`:
- `useDetectMobileOrTablet` -> uses screen width
- `isIos` and `isAndroid` is provided by `capacitor`

**Review by MattiaPrimavera — CHANGES_REQUESTED** [review]

**Comment by MattiaPrimavera** [comment]
Needs to be rebased

**Comment by MattiaPrimavera** [comment]
pls consider reducing the mobile `AppBar` height: in my advice the logo is taking too much space on mobile, thus making the appbar too big

File: src/components/Logo/Logo.tsx (hunk 1 above)
Comment (ntchou) [reply src/components/Logo/Logo.tsx:24]: An oversight. I fix it

**Comment by ntchou** [comment]
> pls consider reducing the mobile `AppBar` height: in my advice the logo is taking too much space on mobile, thus making the appbar too big

If i reduce logo size more we'll don't see `Instant visio`. As you want.

Is it ok ?
image

**Review by MattiaPrimavera — APPROVED** [review]

**Pull request merged**